src/components/screens/NewsItem.js

```diff
@@ -208,13 +208,13 @@ export const NewsItem = ({ data, navigation }) => {
         {!!title && !!link ? (
           <TitleContainer>
             <Touchable onPress={openWebScreen}>
-              <Title>{trimNewLines(title)}</Title>
+              <Title accessibilityLabel={`${title} (Überschrift)`}>{trimNewLines(title)}</Title>
             </Touchable>
           </TitleContainer>
         ) : (
           !!title && (
             <TitleContainer>
-              <Title>{trimNewLines(title)}</Title>
+              <Title accessibilityLabel={`${title} (Überschrift)`}>{trimNewLines(title)}</Title>
```

Comment:
please add the `trimNewLines(title)` also in the `accessibilityLabel`, so we will not have problems with some new lines that can exist